src/Simulation/Common/QubitManager.cs

```diff
@@ -544,7 +547,7 @@ internal IQArray<Qubit> Borrow(long numToBorrow, IEnumerable<Qubit> qubitsInUse)
         {
             var inUse = new Stack<Qubit>(qubitsInUse.Distinct().OrderByDescending(q => q.Id));
             var borrowed = QArray<Qubit>.Create(numToBorrow);
-            long numBorrowed = System.Math.Min(NumAllocatedQubits - inUse.Count, numToBorrow);
+            long numBorrowed = System.Math.Min(NumAllocatedQubits - inUse.Count, numToBorrow);

```

Comment:
Do you know why this line is marked as changed?

Reply:
Line ending. Normalized the file to use "LF" consistently. Thank you for pointing this out!